portfolio/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,199 @@
+
+*{
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+body{
+    background-color: rgb(0, 36, 61);
+    height: 100%;
+    overflow: hidden; /*To hide scroll bar*/
+    line-height: 1.6;
+}
+a{
+    text-decoration: none;
+    color: whitesmoke;
+}
+a:hover{
+    color: rgb(247, 227, 92);
+}
+
+.content{
+    color:whitesmoke;
+    bottom: 0%;
+    width: 960px;
+    margin-left: auto;
+    margin-right:auto;
+    background-color: rgb(0, 36, 61);
+   /* background-color: rgb(43, 72, 107);*/
+    box-shadow: 0px -5px 30px black;
+}
+
+.navbar{
```

Comment:
the coding style guide seems to point out  a space before the starting {.
https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules